normits_demand/models/notem/attraction_models.py

```diff
@@ -39,6 +39,7 @@
 
 
 class HBAttractionModel(HBAttractionModelPaths):
+    _log_fname = "NoTEM_log.log"
```

Comment:
Name the log after the model it's running in.

e.g. HBAttractionModel_log.log

Repeat this for all 4 models